### ISSUE TITLE
[bug]: Sidebar on mobile requires a DialogTitle

### ISSUE THREAD

**Issue by daikiejp** [daikiejp/nextadmin-intl-theme-auth#1]
```
`DialogContent` requires a `DialogTitle` for the component to be accessible for screen reader users.`
```

```
`"Warning: Missing Description or aria-describedby={undefined} for {DialogContent}. Error Component Stack.."`
```

**Comment by daikiejp** [comment]
Bug confirmed at: https://github.com/shadcn-ui/ui/issues/5746

Solutions `sidebar.tsx`:
https://github.com/shadcn-ui/ui/issues/5746#issuecomment-2461248985

and `sheet.tsx`

https://github.com/shadcn-ui/ui/issues/5746#issuecomment-2488124182

**Comment by Eavilesz** [comment]
LGTM
